### ISSUE TITLE
NoSuchFieldError: SPLASH_POTION

### ISSUE THREAD

**Issue by Oruss7** [slipcor/pvparena#217]
When loading arenas `/pa reload`

```
`[11:36:45] [Server thread/ERROR]: null
org.bukkit.command.CommandException: Unhandled exception executing command 'pa' in plugin pvparena v1.3.2.153
at org.bukkit.command.PluginCommand.execute(PluginCommand.java:46) ~[spigot-1.8.8.jar:git-Spigot-e4d4710-e1ebe52]
at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:141) ~[spigot-1.8.8.jar:git-Spigot-e4d4710-e1ebe52]
at org.bukkit.craftbukkit.v1_8_R3.CraftServer.dispatchCommand(CraftServer.java:641) ~[spigot-1.8.8.jar:git-Spigot-e4d4710-e1ebe52]
at net.minecraft.server.v1_8_R3.PlayerConnection.handleCommand(PlayerConnection.java:1162) [spigot-1.8.8.jar:git-Spigot-e4d4710-e1ebe52]
at net.minecraft.server.v1_8_R3.PlayerConnection.a(PlayerConnection.java:997) [spigot-1.8.8.jar:git-Spigot-e4d4710-e1ebe52]
at net.minecraft.server.v1_8_R3.PacketPlayInChat.a(PacketPlayInChat.java:45) [spigot-1.8.8.jar:git-Spigot-e4d4710-e1ebe52]
at net.minecraft.server.v1_8_R3.PacketPlayInChat.a(PacketPlayInChat.java:1) [spigot-1.8.8.jar:git-Spigot-e4d4710-e1ebe52]
at net.minecraft.server.v1_8_R3.PlayerConnectionUtils$1.run(SourceFile:13) [spigot-1.8.8.jar:git-Spigot-e4d4710-e1ebe52]
at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_60]
at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_60]
at net.minecraft.server.v1_8_R3.SystemUtils.a(SourceFile:44) [spigot-1.8.8.jar:git-Spigot-e4d4710-e1ebe52]
at net.minecraft.server.v1_8_R3.MinecraftServer.B(MinecraftServer.java:715) [spigot-1.8.8.jar:git-Spigot-e4d4710-e1ebe52]
at net.minecraft.server.v1_8_R3.DedicatedServer.B(DedicatedServer.java:374) [spigot-1.8.8.jar:git-Spigot-e4d4710-e1ebe52]
at net.minecraft.server.v1_8_R3.MinecraftServer.A(MinecraftServer.java:654) [spigot-1.8.8.jar:git-Spigot-e4d4710-e1ebe52]
at net.minecraft.server.v1_8_R3.MinecraftServer.run(MinecraftServer.java:557) [spigot-1.8.8.jar:git-Spigot-e4d4710-e1ebe52]
at java.lang.Thread.run(Thread.java:745) [?:1.8.0_60]
Caused by: java.lang.NoSuchFieldError: SPLASH_POTION
at net.slipcor.pvparena.core.StringParser.hasPotionMeta(StringParser.java:473) ~[?:?]
at net.slipcor.pvparena.core.StringParser.hasPotionMeta(StringParser.java:469) ~[?:?]
at net.slipcor.pvparena.core.StringParser.getItemStackFromString(StringParser.java:389) ~[?:?]
at net.slipcor.pvparena.managers.ConfigurationManager.configParse(ConfigurationManager.java:210) ~[?:?]
at net.slipcor.pvparena.arena.Arena.(Arena.java:111) ~[?:?]
at net.slipcor.pvparena.managers.ArenaManager.load_arenas(ArenaManager.java:299) ~[?:?]
at net.slipcor.pvparena.PVPArena.onCommand(PVPArena.java:293) ~[?:?]
at org.bukkit.command.PluginCommand.execute(PluginCommand.java:44) ~[spigot-1.8.8.jar:git-Spigot-e4d4710-e1ebe52]
... 15 more
```

**Comment by Oruss7** [comment]
v1.3.3.153 and 128 pa modules

**Comment by CCCCarsten** [comment]
Minecraft SPIGOT 1.8.8 - does newest version of pvparena.jar not only 1.9.x ?

**Comment by slipcor** [comment]
Yes. I have moved to 1.9 !!

Please refer to http://www.slipcor.net/?page=plugins for downloads based on minecraft version :)

**Comment by Oruss7** [comment]
Ah we move 1.9 this monday xD

**Comment by slipcor** [comment]
yeah the fixes that are in place in 1.3.3 should not be critical for 1.8 so you should be fine until then :)

**Comment by Oruss7** [comment]
Use 1.8 versions.
